manifests/init.pp

```diff
@@ -108,6 +108,8 @@
 #
 # $pulpcore_postgresql_ssl_root_ca::    Path to the root certificate authority to validate the certificate supplied by the PostgreSQL database server.
 #
+# $pulpcore_allowed_import_path::       allowed import paths for pulpcore settings.py
+# $pulpcore_allowed_import_path::       allowed export paths for pulpcore settings.py
```

Comment:
If you wish to have some documentation about how these parameters are used in Katello / Content Proxy, you're allowed to be more specific here because this puppet module specifically exists for those use cases

Reply:
Also, you have `$pulpcore_allowed_import_path::` twice here. One should be `export` instead.

Reply:
oops!  Updated